src/llmq/options.cpp

```diff
@@ -31,14 +31,14 @@ static bool EvalSpork(Consensus::LLMQType llmqType, int64_t spork_value)
     return false;
 }
 
-bool IsAllMembersConnectedEnabled(Consensus::LLMQType llmqType)
+bool IsAllMembersConnectedEnabled(const Consensus::LLMQType llmqType, const CSporkManager& sporkman)
 {
-    return EvalSpork(llmqType, sporkManager->GetSporkValue(SPORK_21_QUORUM_ALL_CONNECTED));
+    return EvalSpork(llmqType, sporkman.GetSporkValue(SPORK_21_QUORUM_ALL_CONNECTED));
```

Comment:
`SPORK_21_QUORUM_ALL_CONNECTED` 
do we still need this spork? @UdjinM6 @PastaPastaPasta @ogabrielides 

is it useful anyhow or is it time to just drop it?

Reply:
Disabling (switching to `0`?) helped resolving CL quorum connection issues somehow in the past iirc. We had no issues like that for quite some time but I'm not sure if we actually fixed it 😅